addons/website_slides/tests/test_security.py

```diff
@@ -209,6 +209,18 @@ def test_access_channel_visibility_members(self):
         with self.assertRaises(AccessError):
             self.channel.with_user(self.user_emp).read(['name'])
 
+    @mute_logger('odoo.models', 'odoo.addons.base.models.ir_rule')
+    def test_access_channel_visiblilty_members_as_invited(self):
+        self.channel.write({'visibility': 'members'})
+        self.channel.flush_model()
```

Comment:
You're right actually, flush_recordset is enough here. I went through the few flush_x in the diff and made some change. The visibility must be flushed because of the constraint and compute field depending on it.